tests/behat/features/bootstrap/DrupalCRFeatureContext.php

```diff
@@ -483,4 +483,63 @@ public function iClickOnElement(string $field): void
     $this->getSession()->getPage()->find('css', $field)->click();
   }
 
+  /**
+   * Creates landing page with story row paragraphs with fields title, fundraiser_total,
+   *         fundraiser_copy, fundraiser_img, fundraiser_bg_color, beneficiary_copy, beneficiary_img
+   *
+   * @Given I create a :type( page ) with :title( title) and story row paragraph with following fields:
+   */
+  public function createLandingPageWithStoryRowParagraph( $type, $title,TableNode $fields ) {
+
+    // First, create a landing page node.
+    $node = (object) [
```

Comment:
I don't like this hardcode way to test things... could you reuse any other existing node instead?